File: shouty-cs/Coordinate.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Shouty
{
    public class Coordinate
    {
        private readonly int xCoord;
        private readonly int yCoord;

        public Coordinate(int xCoord, int yCoord)
        {
            this.xCoord = xCoord;
            this.yCoord = yCoord;
        }

        public int DistanceFrom(Coordinate other)
        {
//         TODO: actually calculate distance. I think we need to use pythagoras' theorem?
            return 0;

//            int xDiff = xCoord - other.xCoord;
//            int yDiff = yCoord - other.yCoord;

//            return (int) Math.Sqrt(Math.Pow(xDiff, 2) + Math.Pow(yDiff, 2));

        }
    }
}

File: shouty-java/Coordinate.java
public class Coordinate {
  private int x;
  private int y;

  public Coordinate(int xCoord, int yCoord){
    x = xCoord;
    y = yCoord;
  }

  public int distanceFrom(Coordinate other){
// TODO Actually calculate distance
    return 0;


// Maybe Pythagoras might be able to help...?
//  int xDiff = x - other.x;
//  int yDiff = y - other.y;

//  return (int) Math.sqrt(Math.pow(xDiff, 2) + Math.pow(yDiff, 2));
  }
}